dashboard/app/models/blockly.rb

```diff
@@ -25,6 +25,7 @@
 require 'nokogiri'
 class Blockly < Level
   include SolutionBlocks
+  include LocaleHelper
```

Comment:
A little weird to be including a helper from a model.  Maybe an indication that some of the helper code should also be moved to a model (or Plain Old Ruby Object)?

For `data_t`, you can call `I18n.t` directly.